plugins/poll/db/migrate/20230614041219_delete_duplicate_poll_votes.rb

```diff
@@ -4,11 +4,11 @@ class DeleteDuplicatePollVotes < ActiveRecord::Migration[7.0]
   def up
     execute <<~SQL
       DELETE FROM poll_votes
-      WHERE (poll_id, user_id, updated_at) NOT IN (
-        SELECT poll_id, user_id, MAX(updated_at)
+      WHERE ctid NOT IN (
+        SELECT MIN(ctid)
```

Comment:
I think the `ctid`s are not necessarily in the order of creation, but I do not think that is a problem.

Reply:
I'm using `created_at` now.